tests/test_gunicorn.py

```diff
@@ -29,6 +29,13 @@
 
 from builtins import *  # noqa
 
+import pytest
+
+try:
+    import gunicorn  # noqa
+except ImportError:
+    pytest.skip("skipping gunicorn only tests", allow_module_level=True)
+
```

Comment:
how about having an `else` clause that deletes the import?
```suggestion
else:
     del gunicorn

```